File: EstructurasControl/Condicionales/Anidadas/Python/ejercicio2.py
print("=== Verificación de voto ===")

edad = int(input("Ingresa tu edad: "))

# Condicional anidada
if edad >= 0:
    if edad >= 18:
        print("Puedes votar, eres mayor de edad.")
    else:
        print("No puedes votar, eres menor de edad.")
else:
    print("La edad ingresada no es válida.")

File: EstructurasControl/Condicionales/Anidadas/Csharp/ejercicio2.cs
using System;

class ProgramaAnidado2
{
    static void Main()
    {
        Console.Write("Ingresa tu nota (0 - 20): ");
        int nota = Convert.ToInt32(Console.ReadLine());

        // Condicional anidada
        if (nota >= 0 && nota <= 20)
        {
            if (nota >= 11)
            {
                Console.WriteLine("¡Aprobaste el curso!");
            }
            else
            {
                Console.WriteLine("Desaprobaste el curso.");
            }
        }
        else
        {
            Console.WriteLine("La nota ingresada no es válida.");
        }
    }
}
